recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "xgboost" %}
 {% set version = "2.1.1" %}
 {% set build_number = 2 %}
-{% set min_python = "3.8" %}
+{% set min_python = "3.9" %}
```

Comment:
This needed to be bumped to Python 3.9 now that this is the minimum Python version in conda-forge